README.md

```diff
@@ -1,9 +1,48 @@
 # Enhanced Traveller Screening
 
-### Covid 19 response app
+### A component of the COVID-19 response
 
 ---
 
+## Introduction
+
+This is a digital service built in partnership with the Ministry of Health and Service B.C. which supports British Columbians returning from out-of-country to self isolate. Travellers complete a self isolation plan which may be reviewed with a provincial official to determine whether they require additional support in order to self isolate. In addition, Service BC is able to followup with travellers to ensure the success of their self isolation plan.
+
+## Table of Contents
+
+1. [Project Status](#project-status)
+1. [Features](#features)
+1. [Getting Help or Reporting an Issue](#getting-help-or-reporting-an-issue)
+1. [How to Contribute](#how-to-contribute)
+1. [Development](#development)
+1. [License](#license)
+
+## Project Status
+
+This application has been released and is in active feature development.
+
+## Features
+
+This project includes the following features:
+
+1. Public web form for returning travellers
+1. Ability to download a PDF version of the form
```

Comment:
There's no ability to download a PDF version of this form at this time.

Reply:
Resolved offline, this references the blank PDF download